reference/forms/types/entity.rst

```diff
@@ -49,7 +49,7 @@ Using a Custom Query for the Entities
 
 If you want to create a custom query to use when fetching the entities
 (e.g. you only want to return some entities, or need to order them), use
-the `query_builder`_ option::
+the `query_builder`_ option. Please note, that `query_builder` is used to compose data, not fetch real final results::
```

Comment:
I am not sure about this change. In the description of the `query_builder` option we have this:

> The value of this option can either be a `QueryBuilder` object, a callable or `null` (which will load all entities). When using a callable, you will be passed the `EntityRepository` of the entity as the only argument and should return a `QueryBuilder`. Returning `null` in the Closure will result in loading all entities.

Maybe change this line to something like this:

```suggestion
the `query_builder`_ option (which must be a `QueryBuilder` object, a closure returning a `QueryBuilder` object or `null` to load all entities)::
```

Reply:
I like a lot this proposal. @toci-to-ci do you like it too? If this is more clear for you now, we can merge this suggestion. Thanks.

Reply:
This is now merged. While merging it I reworded as suggested by @xabbuh.